Source/Shared/Extensions/String+Regex.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  String+Regex.swift
+//  Sugar
+//
+//  Created by Aashish Dhawan on 18/03/16.
+//  Copyright © 2016 Hyper Interaktiv AS. All rights reserved.
+//
+
+import Foundation
+
+public enum Regex: String {
+    
+    case Email = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,6}"
+    case PhoneNumber = "^[0-9]{10}$"
+    case ZipCode = "^[0-9]{5}$"
```

Comment:
I don't know about this one as it will be different depending on country. A zip code in Norway contains 4 digits and in Sweden we use 5 digits. What do you think?